package.json

```diff
@@ -18,45 +18,41 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.2.4",
-    "@angular/cdk": "^15.2.4",
-    "@angular/common": "^15.2.4",
-    "@angular/compiler": "^15.2.4",
-    "@angular/core": "^15.2.4",
-    "@angular/forms": "^15.2.4",
-    "@angular/material": "^15.2.4",
-    "@angular/platform-browser": "^15.2.4",
-    "@angular/platform-browser-dynamic": "^15.2.4",
-    "@angular/router": "^15.2.4",
-    "@angular/service-worker": "^15.2.4",
-    "@ngneat/until-destroy": "^9.2.3",
+    "@angular/animations": "^16.0.0",
+    "@angular/cdk": "~16.0.0",
+    "@angular/common": "^16.0.0",
+    "@angular/compiler": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@angular/forms": "^16.0.0",
+    "@angular/material": "~16.0.0",
+    "@angular/platform-browser": "^16.0.0",
+    "@angular/platform-browser-dynamic": "^16.0.0",
+    "@angular/router": "^16.0.0",
+    "@angular/service-worker": "^16.0.0",
+    "@ngneat/until-destroy": "^10.0.0-beta.0",
```

Comment:
Create a PR to replace it by [takeUntilDestroyed](https://angular.io/api/core/rxjs-interop/takeUntilDestroyed)